tasks/fix-cat2.yml

```diff
@@ -1439,63 +1439,104 @@
-  command: "true"
+  file:
+    dest: "{{ item }}"
+    mode: 0644
+    state: file
+  with_items: "{{ rhel_07_040640_audit.stdout_lines }}"
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040640
 
 - name: "MEDIUM | RHEL-07-040650 | PATCH | The SSH private host key files must have mode 0600 or less permissive."
-  command: "true"
+  file:
+    dest: "{{ item }}"
+    mode: 0600
+    state: file
+  with_items: "{{ rhel_07_040650_audit.stdout_lines }}"
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040650
 
 - name: "MEDIUM | RHEL-07-040660 | PATCH | The SSH daemon must not permit Generic Security Service Application Program Interface (GSSAPI) authentication unless needed."
-  command: "true"
+  lineinfile:
+      dest: /etc/ssh/sshd_config
+      regexp: (?i)^#?gssapiauthentication
+      line: GSSAPIAuthentication no
+      validate: sshd -t -f %s
+  notify: restart ssh
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040660
+      - ssh
 
 - name: "MEDIUM | RHEL-07-040670 | PATCH | The SSH daemon must not permit Kerberos authentication unless needed."
-  command: "true"
+  lineinfile:
+      dest: /etc/ssh/sshd_config
+      regexp: (?i)^#?kerberosauthentication
+      line: KerberosAuthentication no
+      validate: sshd -t -f %s
+  notify: restart ssh
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040670
+      - ssh
 
 - name: "MEDIUM | RHEL-07-040680 | PATCH | The SSH daemon must perform strict mode checking of home directory configuration files."
-  command: "true"
+  lineinfile:
+      dest: /etc/ssh/sshd_config
+      regexp: (?i)^#?strictmodes
+      line: StrictModes yes
+      validate: sshd -t -f %s
+  notify: restart ssh
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040680
+      - ssh
 
 - name: "MEDIUM | RHEL-07-040690 | PATCH | The SSH daemon must use privilege separation."
-  command: "true"
+  lineinfile:
+      dest: /etc/ssh/sshd_config
+      regexp: (?i)^#?useprivilegeseparation
+      line: UsePrivilegeSeparation yes
+      validate: sshd -t -f %s
+  notify: restart ssh
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040690
+      - ssh
 
 - name: "MEDIUM | RHEL-07-040700 | PATCH | The SSH daemon must not allow compression or must only allow compression after successful authentication."
-  command: "true"
+  lineinfile:
+      dest: /etc/ssh/sshd_config
+      regexp: (?i)^#?compression
+      line: Compression no
+      validate: sshd -t -f %s
+  notify: restart ssh
   tags:
       - cat2
       - medium
       - patch
       - RHEL-07-040700
+      - ssh
 
 - name: "MEDIUM | RHEL-07-040730 | PATCH | The system must not be performing packet forwarding unless the system is a router."
-  command: "true"
+  sysctl:
+      name: net.ipv4.ip_forward
+      present: yes
+      value: 0
```

Comment:
Add `ignoreerrors: yes` as well as a conditional to specify whether the system is a router.